modules/ROOT/pages/faq.adoc

```diff
@@ -5,3 +5,17 @@ Why ELN?::
     The Fedora distribution is, among other things, an https://docs.fedoraproject.org/en-US/quick-docs/fedora-and-red-hat-enterprise-linux/[upstream] for
     Red Hat Enterprise Linux (RHEL). By adjusting buildroot and assorted
     configurations, ELN allows Fedora contributors who work on RHEL and other similarly targeted downstreams to judge the state of upstream from their perspective. The benefit for the Fedora distribution is that contributors who work on both upstream and downstream stay more continuously connected to the state of Fedora -- especially, but not exclusively, Rawhide.
+
+What is the role of ELN with respect to CentOS Stream?::
+
+     ELN is a continuous rebuild of Fedora Rawhide sources. Once in a while a
+     certain snapshot of the ELN is used to bootstrap the new CentOS Stream for
+     the new major RHEL release. THe is no continuous synce between ELN and ant
+     version of the CentOS Stream. Bootstrap activity happens only once for each
+     CentOS Stream version. See the picture:
```

Comment:
```suggestion
     ELN is a continuous rebuild of Fedora Rawhide sources. ELN does not
     follow the branching of Fedora into releases. Approximately every three
     years, a stabilization branch of Fedora is selected to bootstrap the next
     major RHEL release. At this time, a new CentOS Stream 9 branch will
     be created and will be synchronized from this Fedora branch until
     Fedora reaches its Final Freeze.
     
     There is no sync between ELN and CentOS Stream at any time. The
     CentOS Stream will essentially *replace* ELN in the synchronization
     process once it becomes available while ELN will continue on to prepare
     for the next eventual RHEL release in a few years.
```

Reply:
>  The CentOS Stream will essentially *replace* ELN in the synchronization process

I think this might be hard to understand for the outsider. You are referencing to the synchronization process, but we don't explain it anywhere.

And since CentOS Stream takes the role of "ELN for Fedora 34" only for a relatively short period of time at the beginning of the CentOS/RHEL development, I wouldn't maybe highlight it as a defining property.